node-repository/src/test/java/com/yahoo/vespa/hosted/provision/autoscale/NodeMetricsDbTest.java

```diff
@@ -18,15 +18,15 @@ public void testNodeMetricsDb() {
         NodeMetricsDb db = new NodeMetricsDb();
         List<NodeMetrics.MetricValue> values = new ArrayList<>();
         for (int i = 0; i < 40; i++) {
-            values.add(new NodeMetrics.MetricValue("host0", "cpu.util", clock.instant().toEpochMilli(), 0.9f));
+            values.add(new NodeMetrics.MetricValue("host0", "cpu.util", clock.instant().getEpochSecond(), 0.9f));
             clock.advance(Duration.ofHours(1));
         }
         db.add(values);
 
-        assertEquals(30, db.getWindow(clock.instant().minus(Duration.ofHours(30)), Resource.cpu,    List.of("host0")).measurementCount());
+        assertEquals(29, db.getWindow(clock.instant().minus(Duration.ofHours(30)), Resource.cpu,    List.of("host0")).measurementCount());
```

Comment:
Why did this happen?

Reply:
We're now going through an API which uses seconds which round the current time to the nearest thousand, which causes the window to end up right at the boundary of the 30th value.

Reply:
Oh, so there was 1/1000 chance this test would fail before? If it ran exactly on the second?

Reply:
Yes, I think so. "Problem" here is I don't really care precisely how many ends up in the window ..